Oxide.Core/RemoteConsole/RemoteConsole.cs

```diff
@@ -156,7 +160,7 @@ private void OnMessage(MessageEventArgs e)
                 case "global.unban":
                     if (args.Length == 0)
                         return;
-                    if (Interface.Call<bool?>("OnRconUnban", args) != false)
+                    if (Interface.CallHook("OnRconUnban", args) != null)
```

Comment:
Interface.Call is fine, no need to use CallHook. The casing removal is fine though.

I'm not really sure that an entirely new hook needs to be made though, as you could use the same Covalence hooks for this so that plugins do not have to listen to multiple hooks.